R/github.R

```diff
@@ -93,3 +112,8 @@ github_DESCRIPTION <- function(username, repo, subdir = NULL, ref = "master", ho
     base64_decode(gsub("\\\\n", "", fromJSONFile(tmp)$content))
   }
 }
+
+github_error_message <- function(res) {
+  msg <- fromJSON(rawToChar(res$content))
```

Comment:
I would put  `simplifyVector  = FALSE` here, to be on the safe side.

Reply:
This is the internal fromJSON from https://github.com/r-lib/remotes/blob/fb6682b6091412e75e313345ab54e96e82ed58b7/R/json.R#L31, so I guess it never simplifies.

Reply:
Oh, right, I wrote that, I remember now.  :)